src/project/models.py

```diff
@@ -3,8 +3,8 @@
 
 class Project(models.Model):
     name = models.CharField(max_length=255)
-    project_number = models.BigIntegerField(blank=True, null=True)
-    old_project_numbers = models.CharField(max_length=255)
+    project_number = models.BigIntegerField()
+    old_project_numbers = models.CharField(max_length=255,blank=True, null=True)
```

Comment:
```suggestion
    old_project_numbers = models.CharField(max_length=255, blank=True, null=True)
```
Stukje code formatting

Reply:
hier loopt de build nu op stuk, je kan hier gelijk deze suggestie doorvoeren